### DISCUSSION TITLE
Unknown error in `create_highlight` func

### DISCUSSION THREAD

**Discussion by Asylcreek** [pwntester/octo.nvim#454] [Q&A]
Is it just me or is this a bug? 

Because I get this error when I try to view PRs with other people involved. 

When I open PRs with just me, it opens just fine ... can anybody help?
<img width="1634" alt="Screenshot 2023-11-13 at 19 48 54" src="https://github.com/pwntester/octo.nvim/assets/54318933/8a47e87c-a353-4040-93b2-c2b80740c234">

**Comment by williambdean** [comment]
Closing this. Please raise a bug if this is still an issue
